### DISCUSSION TITLE
[Support] k8s 使用 bootstrap 设置 worker数

### DISCUSSION THREAD

**Discussion by MotoLine** [midwayjs/midway#3713] [Q&A]
### Describe the problem（描述问题）

@czy88840616 在K8S中使用 bootstrap 进行部署的时候 可以设置多个worker吗？我看egg-scripts 可以使用 --worker 来设置，bootstrap 是否提供了相关配置？ 

### Midway Versions（Midway 版本）

✓ @midwayjs/async-hooks-context-manager(3.15.0)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.15.0)
✓ @midwayjs/consul(3.15.1)
✓ @midwayjs/core(3.15.0)
✓ @midwayjs/decorator(3.15.0)
✓ @midwayjs/http-proxy(3.15.1)
✓ @midwayjs/i18n(3.15.1)
✓ @midwayjs/mock(3.15.0)
✓ @midwayjs/static-file(3.15.1)
✓ @midwayjs/validate(3.15.1)
✓ @midwayjs/view(3.15.1)
✓ @midwayjs/view-nunjucks(3.15.1)
✓ @midwayjs/web(3.15.0)

**Comment by czy88840616** [comment]
https://midwayjs.org/docs/extensions/socketio#%E8%B0%83%E6%95%B4%E8%BF%9B%E7%A8%8B%E6%A8%A1%E5%9E%8B 参考这里，使用 ClusterManager
